x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/rules/action/DeleteQueryRulesetAction.java

```diff
@@ -29,7 +29,7 @@
 public class DeleteQueryRulesetAction extends ActionType<AcknowledgedResponse> {
 
     public static final DeleteQueryRulesetAction INSTANCE = new DeleteQueryRulesetAction();
-    public static final String NAME = "cluster:admin/xpack/query_rules/delete";
+    public static final String NAME = "cluster:admin/search/query_rules/delete";
```

Comment:
why did we change this?

Reply:
I renamed this based on the discussion we had to make a manage search privilege that these fall under. If we don't want to do this I can change it back to `xpack`. WDYT?

Reply:
As mentioned in https://github.com/elastic/elasticsearch/pull/97853, I don't think that's necessary. I also don't see a common grouping like that in the rest of the actions. I'd vote for keeping as is unless we have a good reason not to.

Reply:
OK, I'll change them back. My bad, I misunderstood the goal.

Reply:
On second thought, should it be `xpack/search/query_rules`?

Reply:
I'd still say that we don't need the /search/ prefix. I find them more descriptive without it. Not a strong opinion though.